packages/manager/apps/telecom/src/app/telecom/pack/pack.bundle.js

```diff
@@ -46,8 +46,8 @@ import './slots/exchangeIndividual/pack-exchangeIndividual.controller';
 import './slots/exchangeLite/pack-exchangeLite.controller';
 import './slots/hostedEmail/add/pack-hostedEmail-add.controller';
 import './slots/hostedEmail/add/pack-hostedEmail-add';
-import './slots/hostedEmail/detail/pack-hostedEmail-detail.controller';
-import './slots/hostedEmail/detail/pack-hostedEmail-detail';
+import './slots/hostedEmail/detail/detail.controller';
```

Comment:
Already imported in `packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/detail.component.js`(https://github.com/ovh/manager/pull/2629/files#diff-cd816964962d819ad20ae16de702bdb8R1)